duetector/tracers/base.py

```diff
@@ -80,6 +80,14 @@ def _convert_data(self, data) -> NamedTuple:
             v = getattr(data, k)
             if isinstance(v, bytes):
                 v = v.decode("utf-8")
+            elif k == "saddr" or k == "daddr":
+                dq = b""
+                for i in range(0, 4):
+                    dq = dq + str(v & 0xFF).encode()
+                    if i != 3:
+                        dq = dq + b"."
+                    v = v >> 8
+                v = dq
```

Comment:
This method of `BccTracer` is intended to transform uncertain bcc output into a deterministic `data_t` (`NamedTuple`), and is best overridden if there is a need for other conversions.

```python
def _convert_data(self, data) -> NamedTuple:
    data = super()._convert_data(data)
    # Then process data
    data.saddr = ...
    data.daddr = ...
    return data
```

Reply:
Thank you foryour  suggestion，I will modify it later.